src/app/legacy/psammead/psammead-navigation/src/index.jsx

```diff
@@ -64,7 +63,7 @@ const ListItemBorder = `
 const StyledLink = styled.a`
   ${({ script }) => script && getPica(script)};
   ${({ service }) => getSansRegular(service)};
-  ${({ brandForegroundColour }) => `color: ${brandForegroundColour};`}
+  color: ${props => props.theme.palette.BRAND_FOREGROUND};
   color: ${C_GREY_10};
```

Comment:
There are two delcarations of 'color': here. Should I remove the one coming from the Brand Palette?

Reply:
Strange one. The `BRAND_FOREGROUND` is typically `GHOST`, which is almost white, whereas `C_GREY_10` is almost black. Almost-black looks like the right colour use here as these are the top navigation link items. 

I would be inclined to use `GREY_10` from the Theme palette to maintain consistency.

Reply:
Fab thanks - I have updated